openhtf/output/web_gui/__init__.py

```diff
@@ -80,7 +80,6 @@
              default_value=[],
              description='List of manually declared stations.')
 
-
 Hostport = collections.namedtuple('Hostport', ['host', 'port'])
 
 
```

Comment:
Yeah, but if you don't properly listen to those complete state objects, then you're still missing updates. If a test sends an update then sits waiting for a prompt, but the frontend missed the update to show the prompt, then we're in no better shape.